Formula/neovim.rb

```diff
@@ -98,7 +98,8 @@ class Neovim < Formula
     end
   end
 
-  option "with-release", "Compile in release mode"
+  option "with-dev", "Compile a Dev build. Enables debug information, logging,
+        and optimizations that don't interfere with debugging."
```

Comment:
are users using `with-release` going to get an error? might want to leave in support for that.